python/paddle/fluid/tests/unittests/mkldnn/test_requantize_mkldnn_op.py

```diff
@@ -29,25 +31,27 @@ def setUp(self):
         self.data_type = 'int8'
         self.set_scale()
         self.set_data_type()
+        self.prepare_inputs()
```

Comment:
Hi, please change this test input_size to [1, 1, 10, 10] too, otherwise it may fail on some CI. https://github.com/PaddlePaddle/Paddle/wiki/OP-Test-Input-Shape-Requirements-(English-Version)

Reply:
Ok, I've changed it.

Reply:
LGTM.